server/service/client_profiles.go

```diff
@@ -96,7 +96,8 @@ func (c *Client) AddProfile(teamID uint, configurationProfile []byte) (uint, err
 	return addProfileResponse.ProfileID, nil
 }
 
-func (c *Client) GetConfigProfilesSummary(teamID *uint) (*fleet.MDMAppleConfigProfilesSummary, error) {
+func (c *Client) GetConfigProfilesSummary(teamID *uint) (*fleet.MDMProfilesSummary, error) {
+	// TODO: Update this to use the new endpoint?
```

Comment:
Yeah I think it should use the new one, the old is now deprecated. (https://github.com/fleetdm/fleet/pull/14230/files#diff-7246bc304b15c8865ed8eaa205e9c244d0a0314e4bae60cf553dc06147c38b64R4174)